src/i18n/en.yaml

```
@@ -18,9 +18,10 @@ home:
     empty: No words
     jotting: '{count} jottings'
     note: '{count} notes'
-  newest: Newest Note
+  latest: Latest
   prefaces: Prefaces
 jotting:
+  noEntries: No jottings available
```

Prefer `empty` as the key.
```suggestion
  empty: No jottings available
```